packages/eslint-config-sentry-app/package.json

```diff
@@ -22,7 +22,7 @@
   },
   "peerDependencies": {
     "eslint": ">=5",
-    "prettier": "1.7.4"
+    "prettier": "1.16.4"
```

Comment:
Should we give a range so that we don't have to deal with conflicting pins in applications?

Reply:
I think we should enforce a single version because we could have conflicting formatting, especially since we generate eslint errors based on prettier formatting.